opm/autodiff/BlackoilModelEbos.hpp

```diff
@@ -298,17 +304,21 @@ namespace Opm {
                 updateState(x,reservoir_state);
                 wellModel().updateWellState(xw, well_state);
 
-                // since the solution was changed, the cache for the intensive quantities
-                // are invalid
-                ebosSimulator_.model().invalidateIntensiveQuantitiesCache(/*timeIdx=*/0);
+                // since the solution was changed, the cache for the intensive quantities are invalid
+                // ebosSimulator_.model().invalidateIntensiveQuantitiesCache(/*timeIdx=*/0);
```

Comment:
As far as I can see, the `if ()` below should be replaced by:

```
invalidateIntensiveQuantitiesCache_ = true;
```

right before `updateState()` (also, `invalidateIntensiveQuantitiesCache_` needs be set to `false` above)?